stags/src/main/scala/co/pjrt/stags/TagGenerator.scala

```diff
@@ -1,13 +1,13 @@
 package co.pjrt.stags
 
 import java.io.File
-
 import scala.meta._
 
 import co.pjrt.stags.paths.Path
 
 object TagGenerator {
 
+
```

Comment:
What's with all the empty new lines?

Reply:
Sorry, the empty lines are remnants of some debug harness that I removed; but didn't catch the empty lines.